### PR TITLE
798 census individual schema final content update

### LINKED ISSUE #663: Census: Individual section Navigation is wrong when no option is selected for 5. Do you stay at another address for more than 30 days a year?
### Expected behaviour
Must take the user to Question 7

### Actual behaviour
Application is navigating to question 6 which is not correct.

### Steps to reproduce the behaviour
Launch Census Schema
Navigate to Individual section
Do not select any option for Q: 5. Do you stay at another address for more than 30 days a year?

### PR DESCRIPTION
**Pull request by benjefferies**
### What is the context of this PR?
Review/Update individual schema
https://trello.com/c/PrZli0nM/798-census-individual-schema-final-content-update

Fixes #663

### How to review
Open sides in one screen and individual survey in the other and check slides

### CODE REVIEW

File: app/data/census_household.json [hunk 1]
```diff
@@ -3708,16 +3855,16 @@
...
                             "id": "business-name-section",
-                            "number": "36",
+                            "number": "39",
                             "title": "{{ [answers.first_name[group_instance], answers.last_name[group_instance]] | format_household_name }}",
                             "questions": [
                                 {
                                     "id": "business-name-question",
-                                    "title": "What is (was) the name of the organisation or business you work (worked) for?",
-                                    "number": "36a",
+                                    "title": "In your main job, What is (was) the name of the organisation or business you work (worked) for?",
```
Comment (ajmaddaford) [inline app/data/census_household.json]: Should this be 'what' rather that 'What'?

File: app/templates/partials/questions/general.html [hunk 2]
```diff
@@ -18,7 +18,12 @@
 
         <legend class="question__legend">
           {%- if question_title -%}
-            <span class="question__title neptune">{{question_title}}</span>
+            <span class="question__title neptune">
+                {%- if question_number -%}
+                    <span>{{question_number}}. </span>
+                {% endif %}
+                {{question_title}}
+            </span>
```
Comment (ajmaddaford) [inline app/templates/partials/questions/general.html:26]: There are `question_titles` and `question_guidance` jinja variables that we should use if possible

**Review by ajmaddaford — CHANGES_REQUESTED** [review]

**Comment by iwootten** [comment]
Is there any reason I'd be getting a confirmation screen with a list with a single item on it in this branch?

screen shot 2016-12-12 at 11 05 49

File: app/templates/confirmation.html [hunk 3]
```diff
@@ -7,8 +7,11 @@
 <a class="u-dib" href="{{previous_location}}">Previous</a>
 
 <div class="header">
-  <h1 class="header__title saturn">Confirmation</h1>
-  <h2 class="header__subtitle">Thank you for your answers, do you wish to submit</h2>
+  <h1 class="header__title saturn">You’ve completed the 2017 Census Test</h1>
+  <h2 class="header__subtitle">Submission</h2>
+    <ul>
```
Comment (iwootten) [inline app/templates/confirmation.html]: Is this really meant to be here? Seems like the confirmation page is being updated in this PR: https://github.com/ONSdigital/eq-survey-runner/pull/688/files, and that's different to this?

**Comment by collisdigital** [comment]
@benjefferies Can you update the Date of Birth example to 1980 instead of 2014 - Discussed with Louise.

**Review by iwootten — DISMISSED** [review]

File: app/data/census_household.json (hunk 1 above)
Comment (benjefferies) [reply app/data/census_household.json]: Done

File: app/templates/partials/questions/general.html (hunk 2 above)
Comment (benjefferies) [reply app/templates/partials/questions/general.html:26]: Pulled in from https://github.com/ONSdigital/eq-survey-runner/pull/683/files

**Comment by benjefferies** [comment]
@collisdigital Changed the date of birth examples
@iwootten this has been fixed in https://github.com/ONSdigital/eq-survey-runner/pull/688/files

File: app/data/census_individual.json [hunk 4]
```diff
@@ -878,7 +882,7 @@
                                                     "value": "Scottish"
                                                 },
                                                 {
-                                                    "label": "Northern Irish",
+                                                    "label": "Northern Iri1sh",
```
Comment (samiwel) [inline app/data/census_individual.json]: Typo?

**Review by samiwel — CHANGES_REQUESTED** [review]

File: app/data/census_individual.json [hunk 5]
```diff
@@ -1922,15 +1940,12 @@
...
+                                                    "value": "Other"
                                                 },
                                                 {
-                                                    "label": "Other (please specify)",
-                                                    "value": "Other",
-                                                    "other": {
-                                                        "label": "Please specify other passports held"
-                                                    }
+                                                    "label": "None",
+                                                    "value": "None"
```
Comment (samiwel) [inline app/data/census_individual.json:2115]: See https://github.com/ONSdigital/eq-survey-runner/pull/691#issuecomment-266376207 I think these need to be swapped so Other is last.

File: app/data/census_individual.json (hunk 4 above)
Comment (benjefferies) [reply app/data/census_individual.json]: Done

File: app/data/census_individual.json (hunk 5 above)
Comment (benjefferies) [reply app/data/census_individual.json:2115]: Spoken to @LJBabbage and this doesn't seem to be an issue

Comment (samiwel) [reply app/data/census_individual.json:2115]: Okay, fair enough, just highlighting it may need to be tested to ensure it works correctly. 👍

**Comment by LJBabbage** [comment]
Referencing Trello card
3. What is date of birth? Example of date done in census_household.json, rather than census_individual.json, probably should be both.
22a. Last year address, seeing 22
23. Other isn't type 'Other' so this doesn't need changing, additional this won't be a problem anyway

File: app/data/census_individual.json [hunk 6]
```diff
@@ -2183,16 +2259,16 @@
...
                             "questions": [
                                 {
                                     "id": "volunteering-question",
                                     "title": "Thinking of the last 12 months, have you taken part in any volunteering for any groups, clubs or organisations?",
-                                    "number": "25",
+                                    "number": "27",
                                     "guidance": [
                                         {
-                                            "description": "Exclude any Court ordered activities"
+                                            "description": "<strong>Exclude</strong> any Court ordered activities"
```
Comment (samiwel) [inline app/data/census_individual.json]: Court should be court

File: app/data/census_individual.json [hunk 7]
```diff
@@ -2233,17 +2309,17 @@
...
                                 {
                                     "id": "employment-type-question",
                                     "title": "Last week were you:",
-                                    "number": "26",
+                                    "number": "28",
                                     "description": "Select all that apply",
                                     "guidance": [
                                         {
-                                            "description": "Include any paid work, including casual or temporary work, even if only for one hour"
+                                            "description": "<strong>Include</strong> any paid work, including casual or temporary work, even if only for one hour"
```
Comment (samiwel) [inline app/data/census_individual.json:2469]: From the google doc it looks like the whole thing should be bold. But there's inconsistency here across questions so, pinch of salt..

File: app/data/census_individual.json [hunk 8]
```diff
@@ -2610,14 +2761,22 @@
...
+                                    "number": "38",
+                                    "guidance": [
+                                        {
+                                            "list": [
+                                                "For example, <b>primary education, repairing cars, contract catering, computer servicing</b>",
+                                                "If you are (were) a civil servant, write <b>government</b>",
+                                                "If you are (were) a local government officer, write local government officer, write <b>local government</b> and give the name of the department within the local authority"
+                                            ]
+                                        }
+                                    ],
```
Comment (samiwel) [inline app/data/census_individual.json]: The google doc does not have a label for the TextArea field.

File: app/data/census_individual.json [hunk 9]
```diff
@@ -2582,13 +2733,13 @@
...
                             "id": "job-description-section",
-                            "number": "34",
+                            "number": "37",
                             "title": "{{ [answers.first_name, answers.last_name] | format_household_name }}",
                             "questions": [
                                 {
                                     "id": "job-description-question",
                                     "title": "Briefly describe what you do (did) in your main job.",
-                                    "number": "34",
+                                    "number": "37",
```
Comment (samiwel) [inline app/data/census_individual.json:2875]: The google doc does not have a label for the TextArea answer.

File: app/data/census_individual.json (hunk 8 above)
Comment (samiwel) [inline app/data/census_individual.json:2911]: Google doc does not have a label for the TextArea answer.

**Comment by benjefferies** [comment]
3. What is date of birth? Example of date done in census_household.json, rather than census_individual.json, probably should be both. - Fixed
22a. Last year address, seeing 22 - Fixed on census household branch
23. Other isn't type 'Other' so this doesn't need changing, additional this won't be a problem anyway Thanks for confirming

File: app/data/census_individual.json (hunk 9 above)
Comment (benjefferies) [reply app/data/census_individual.json:2875]: I think we agreed to keep this on the census household so I have sync'd with what that defines in the label is that correct @jonnyshaw89?

File: app/data/census_individual.json (hunk 8 above)
Comment (benjefferies) [reply app/data/census_individual.json:2911]: See other comment about label

File: app/data/census_individual.json (hunk 6 above)
Comment (benjefferies) [reply app/data/census_individual.json]: Done

File: app/data/census_individual.json (hunk 7 above)
Comment (benjefferies) [reply app/data/census_individual.json:2469]: Think it might be an error in the slide

**Review by samiwel — APPROVED** [review]

**Comment by ajmaddaford** [comment]
Tested against latest slides. Found:
- Some guidance text gets cut off on the right hand side:
screen shot 2016-12-12 at 21 43 57
- Question numbers aren't being output for all questions.
- Routing issue on question '23. What passports do you hold?' - If other selected with other options, 23a is not displayed. This needs #687 merged before it can work

**Comment by collisdigital** [comment]
@ajmaddaford  Question numbers being output on questions with multiple answers is fixed in #683 see https://github.com/ONSdigital/eq-survey-runner/pull/683/files#diff-e80db9608eb2273765145ab36a20454eR23

**Review by ajmaddaford — APPROVED** [review]

**Pull request merged**